index.html

```diff
@@ -4853,6 +4853,67 @@ <h3>Issuer</h3>
         </p>
       </section>
 
+      <section class="informative">
+        <h4>Holder</h4>
+        <p>
+The value associated with the <code>holder</code> <a>property</a> is expected
+to identify the <a>holder</a> to the <a>verifier</a>.
```

Comment:
```suggestion
to be usable to identify the <a>holder</a> to the <a>verifier</a>.
```

The identifier used may not be known to the verifier prior to the creation of this VP. In that case it does not yet identify anything for the verifier.